tests/dtm/test_rgw_s3_process_restart.py

```diff
@@ -777,3 +772,71 @@ def test_overwrite_same_object_during_rgw_restart(self):
         assert_utils.assert_true(resp[0], resp[1])
         self.test_completed = True
         self.log.info("ENDED: Overwrite same object during m0d restart.")
+
+    @pytest.mark.lc
+    @pytest.mark.dtm
+    @pytest.mark.tags("TEST-42252")
+    def test_io_operations_before_after_rgw_restart(self):
+        """Verify IO operations work after rgw restart using pkill"""
+        self.log.info("STARTED: Verify IO operations work after rgw restart using pkill")
+        test_cfg = DTM_CFG['test_42252']
+        test_prefix = 'test-42252-before-restart'
+
+        self.log.info("Step 1: Perform WRITEs/READs-Verify/DELETEs with variable sizes objects.")
+        resp = self.ha_obj.ha_s3_workload_operation(s3userinfo=self.iam_user,
+                                                    log_prefix=test_prefix,
+                                                    nclients=test_cfg['nclients'],
+                                                    nsamples=test_cfg['nsamples'])
+        assert_utils.assert_true(resp[0], resp[1])
+        self.log.info("Step 1: Successfully performed WRITEs/READs-Verify/DELETEs with variable "
+                      "sizes objects.")
+
+        self.log.info("Step 2: Perform Single rgw Process Restart")
+        resp = self.dtm_obj.process_restart(master_node=self.master_node_list[0],
+                                            health_obj=self.health_obj,
+                                            pod_prefix=const.SERVER_POD_NAME_PREFIX,
+                                            container_prefix=const.RGW_CONTAINER_NAME,
+                                            process=self.rgw_process, check_proc_state=True)
+        assert_utils.assert_true(resp, "Failure observed during process restart/recovery")
+        self.log.info("Step 2: rgw restarted and recovered successfully")
+
+        test_prefix = 'test-42252-after-restart'
```

Comment:
creating new bucket will be supported after rgw restart, just confirming once again.

Reply:
Yes bucket creation will be supported after rgw restart.